### PR TITLE
Updated the Farsi translation

### PR DESCRIPTION
**Pull request by sudo-arash**
- [x] Have you followed the [contributing guidelines](https://github.com/github/opensource.guide/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you explained what your changes do, and why they add value to the Guides?

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

-----

I updated the Farsi translation and made it better.

### CODE REVIEW

**Review by ratchanonsuttawas — COMMENTED** [review]
Good

**Review by tomthorogood — APPROVED** [review]

**Comment by Datis77re** [comment]
> Good

**Review by ratchanonsuttawas — COMMENTED** [review]
VIO V15PRO

**Comment by ratchanonsuttawas** [comment]
> - [x] Have you followed the [contributing guidelines](https://github.com/github/opensource.guide/blob/HEAD/CONTRIBUTING.md)?
> - [x] Have you explained what your changes do, and why they add value to the Guides?
> 
> **Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**
> 
> -----
> 
> I updated the Farsi translation and made it better.

**Review by ratchanonsuttawas — COMMENTED** [review]
284da256139632215efe8d1cc2f18c5d5a4aba04

**Review by ratchanonsuttawas — COMMENTED** [review]
Approve

File: _data/locales/fa.yml
Comment (ratchanonsuttawas) [inline _data/locales/fa.yml:1]: Approve

Comment (ratchanonsuttawas) [inline _data/locales/fa.yml:1]: Requests change

**Review by ratchanonsuttawas — COMMENTED** [review]
Approve review

**Pull request merged**